.github/workflows/cli-integration.yml

```diff
@@ -0,0 +1,51 @@
+name: 'Waiter cli-integration tests'
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6.x'
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install Python Dependencies
+        run: |
+          cd cli
+          pip install -e .
+          pip install -r integration/requirements.txt
+      - name: Run cli-integration tests
+        run: |
+          cd cli
+          ./bin/ci/run-integration-tests.sh
+        env:
+          GDRIVE_LOG_POST_URL: ${{ secrets.GDRIVE_LOG_POST_URL }}
```

Comment:
Using a secret for this value means it won't work for PRs opened from our forks.

It looks like there are ways to get GitHub to trigger builds with access to secrets even if they came from a fork — but it's not automatic. The example I found required a slash command:

https://github.com/imjohnbo/ok-to-test

We could probably have something where we have a list of core devs that should have builds with secrets.

However, I'm not sure it's worth the effort. I set up this URL specifically because it's low risk to expose. I think it's fine to configure it like we did in Travis.

Reply:
Are you suggesting we explicitly put the url in the configuration yaml files?

Reply:
I think on Travis we were able to configure it as an environment variable in their UI — but GitHub doesn't seem to have anything like that, so I guess sticking it in the YAML is the best option.